IdentityCore/src/MSIDError.h

```diff
@@ -282,6 +282,8 @@ extern NSError * _Nonnull MSIDCreateError(NSString * _Nonnull domain, NSInteger
 
 extern MSIDErrorCode MSIDErrorCodeForOAuthError(NSString * _Nullable oauthError, MSIDErrorCode defaultCode);
 
+extern MSIDErrorCode MSIDErrorCodeForOAuthErrorWithSubErrorCode(NSString * _Nullable oauthError, MSIDErrorCode defaultCode, NSString * _Nullable subError);
```

Comment:
Can we add unit tests for it?

Reply:
The tests to this is going to be in the Broker, same as the original method's tests are.

Reply:
It does make more sense for the unit tests for this function to live in common core.